test/integration/consul-container/libs/node/node.go

```diff
@@ -8,10 +8,13 @@ import (
 
 // Node represent a Consul node abstraction
 type Node interface {
```

Comment:
At this point I think the name `Node` became confusing, maybe ConsulNode or even Consul?

Reply:
yes Agent is good!